spec/unit/mixin/oci_metadata_spec.rb

```diff
@@ -49,7 +49,7 @@
       http_mock = double("http", { code: "404" })
       allow(mixin).to receive(:http_get).and_return(http_mock)
 
-      expect(mixin.logger).to receive(:warn)
+      expect(mixin.logger).to receive(:debug)
```

Comment:
Mike and I fixed this spec test. We want to have the log message for debug and not do a warn as it is confusing to see that warning that looks like a failure when volumes endpoint is expected not to exist unless you configure the cloud instance to have a special high performance volume. This is creating confusion for some users and moving to debug makes sense for troubleshooting and to remain silent in normal operation.